File: MessManagingCalculator/app/src/main/java/com/example/ovi/messmanagingcalculator/MainActivity.java
package com.example.ovi.messmanagingcalculator;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button startBtn;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        startBtn= (Button) findViewById(R.id.startBtn);
        startBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent i=new Intent(MainActivity.this,Second.class);
                startActivity(i);
            }
        });
    }
}

File: DisplayCalendarEvent/app/src/main/java/com/hisoft/ovi/displaycalendarevent/MainActivity.kt
package com.hisoft.ovi.displaycalendarevent

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import java.util.*

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var button = findViewById(R.id.dispbut) as Button
        button.setOnClickListener { disp() }
    }

    fun disp() {
        val startTime = Calendar.getInstance()
        startTime.set(2013, 2, 13, 11, 35)
        val uri = Uri.parse("content://com.android.calendar/time/" + startTime.timeInMillis.toString())
        val intent = Intent(Intent.ACTION_VIEW, uri)
        // Use the Calendar app to view the time.
        startActivity(intent)
    }
}
